pointcloud/testutils.go

```diff
@@ -0,0 +1,30 @@
+package pointcloud
+
+import (
+	"github.com/golang/geo/r3"
+)
+
+// MakeTestPointCloud creates a test point cloud with 3 points.
+func MakeTestPointCloud(label string) *BasicOctree {
```

Comment:
same thing here this should not become a public function.  I would also like to avoid the creation of a new file, so lets just stick it in whichever test.go file is appropriate